Sources/BraintreeThreeDSecure/BTThreeDSecureV2Provider.swift

```diff
@@ -122,14 +122,20 @@ extension BTThreeDSecureV2Provider: CardinalValidationDelegate {
                 completion: completionHandler
             )
         case .error, .timeout:
-            let userInfo = [NSLocalizedDescriptionKey: validateResponse.errorDescription]
+            var userInfo = [NSLocalizedDescriptionKey: validateResponse.errorDescription]
```

Comment:
Can we remove `userInfo` and `errorCode` since we're not using them?

Reply:
`userInfo` and `errorCode` get used within the case so we'll need to keep them... I can move `errorCode` as a global variable. Since I'm splitting off `.error` and `.timeout`, I'll keep `userInfo` in as local variable